src/expandableCalendar/week.tsx

```diff
@@ -68,20 +72,13 @@ const Week = (props: WeekProps) => {
     return week;
   };
 
-  const partialWeekStyle = useMemo(() => {
-    return [style.current.partialWeek, {paddingLeft: timelineLeftInset}];
-  }, [timelineLeftInset]);
-
   return (
     <View style={style.current.container}>
       <View style={[style.current.week, numberOfDays > 1 ? partialWeekStyle : undefined, propsStyle]}>{renderWeek()}</View>
     </View>
   );
-};
+}, isEqual);
```

Comment:
I see you removed your deep comparison of the `merkedDates` and `context` props, but 'isEqual' will not deep check them now and will only compaier their ref (it will deep check prop object only). Is this what you mean to do?

Reply:
according to the docs isEqual is deep comparison
<img width="1392" alt="Screen Shot 2022-09-12 at 10 05 30" src="https://user-images.githubusercontent.com/49347352/189592900-207ae9f2-91b0-4f5f-81be-0713f7d692ce.png">